x/ibc/20-transfer/genesis.go

```diff
@@ -7,8 +7,24 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/ibc/20-transfer/types"
 )
 
+type GenesisState struct {
```

Comment:
Simple GenesisState that for now, is only being used to force the module manager to run `InitGenesis` of transfer module so we can bind to the port

Reply:
can you add that comment on top of the struct?